extensions/coffee/src/toggleCaffeinate.ts

```diff
@@ -0,0 +1,14 @@
+import { runAppleScript } from "run-applescript";
+import Caffeinate from "./caffeinate"
+import Decaffinate from "./decaffeinate"
+
+const ToggleCaffeinate = async () => {
+  try {
+    await runAppleScript(`do shell script "pgrep caffeinate"`);
```

Comment:
I realize this is redundant to what happens in both caffeinate/decaffeinate, but making everything super DRY seems more convoluted than necessary. But, if you support this PR but the redundancy removed, I'm happy to do it.